local_dev/compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.10"
+name: globus-compute
+services:
+
+  endpoint:
+    build:
+      context: ../
+      dockerfile: endpoint_dev.Dockerfile
+      args:
+        LOCAL_USER: ${LOCAL_USER:-${USER}}
+    volumes:
+      - ../:/opt/globus_compute
```

Comment:
In terms of not "over exposing" (... in this private, per-dev, setup!), consider mounting the respective compute directories individually.  Perhaps:
```yaml
    volumes:
      - ../compute_sdk:/opt/globus_compute/compute_sdk
      - ../compute_endpoint:/opt/globus_compute/compute_endpoint
```

Reply:
I mount the entire repo so we can run tests, etc. in the container.